### PR TITLE
[Communication] Update iOS March release

### PR DESCRIPTION
**Pull request by jorgegarchirota**
Add ACS release notes for iOS

### CODE REVIEW

**Comment by jorgegarchirota** [comment]
@RezaJooyandeh , @ankitarorabit  if you can take a look into the iOS release notes

File: releases/2021-03/ios.md [hunk 1]
```diff
@@ -37,13 +40,27 @@ To add the Azure SDK for iOS to your application, follow the example in [Importi
 Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
 
 ```swift
-// Insert dependencies here
+    dependencies: [
+        ...
+        .package(url: "https://github.com/Azure/azure-sdk-for-ios.git", from: "1.0.0-beta.8")
```
Comment (JoshuaLai) [inline releases/2021-03/ios.md:45]: Is this beta 8 or beta 9?

**Review by JoshuaLai — APPROVED** [review]
lgtm

File: releases/2021-03/ios.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ The Azure SDK team is pleased to announce our March 2021 client library releases
 
 #### Beta
 
+- Azure Communication Services Calling
+- Azure Communication Services Chat
+- Azure Communication Services Common
 - _Add packages_
```
Comment (tjprescott) [inline releases/2021-03/ios.md:24]: This line should be removed

File: releases/2021-03/ios.md [hunk 3]
```diff
@@ -74,6 +101,43 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- For parsing of push notification payload `IncomingCallPushNotification` has been renamed to `PushNotificationInfo`.
+- `CallerInfo` class created which provides information about the caller in an incoming call. Can be retrieved from `IncomingCall` and `Call` objects. 
+
+#### Key Bug Fixes
+- `OnCallsUpdated` event is raised when the call collection on `CallAgent` is updated for outgoing calls.
+- `Hold` and `Resume` of an active call is fixed. 
+
 ### _Version_(_Link to Changelog_)
 
 #### _Package name_
```
Comment (tjprescott) [inline releases/2021-03/ios.md:143]: These lines should be removed

**Review by tjprescott — CHANGES_REQUESTED** [review]
Some template boilerplate needs to be removed, but thanks for creating this!

**Review by jorgegarchirota — COMMENTED** [review]
The changes requested were addressed

**Review by tjprescott — APPROVED** [review]

**Comment by tjprescott** [comment]
@czubair if this looks okay to you it should be ready to merge.

File: releases/2021-03/ios.md [hunk 4]
```diff
@@ -74,11 +101,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
-
-#### _Package name_
-
-- Major changes only!
+### 1.0.0-beta.9 ([Changelog](https://github.com/Azure/azure-sdk-for-ios/blob/master/CHANGELOG.md#100-beta9-2021-03-10))
+#### Azure Communication
+##### Breaking Changes
+- Removal of `CommunicationCloudEnvironment.fromModel()` method
+- Removal of label `identifier` in `CommunicationUserIdentifier` and `UnknownIdentifier`
+- `CommunicationIdentifierModel` and `CommunicationIdentifierSerializer` are no longer part of the communication package, they have been moved to AzureCommunicationChat
```
Comment (RezaJooyandeh) [inline releases/2021-03/ios.md:109]: I do not think we should even mention where they are moved. These two types are not for developers to use. Should we say that they are removed from Azure Communication?

File: releases/2021-03/ios.md [hunk 5]
```diff
@@ -74,11 +101,42 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+
+#### Azure Communication Chat
+
+### New Features
+- Introduction of  a new struct `CommunicationIdentifierModel` to repesent a union type that is either a `communicationUser`, `phoneNumber`, or `microsoftTeamsUser`.
```
Comment (RezaJooyandeh) [inline releases/2021-03/ios.md:114]: Let's omit that. This is not a feature for developers.

File: releases/2021-03/ios.md (hunk 4 above)
Comment (jorgegarchirota) [reply releases/2021-03/ios.md]: @RezaJooyandeh  I update the docs ;)

**Review by jorgegarchirota — COMMENTED** [review]
The changes requested were addressed

**Review by RezaJooyandeh — APPROVED** [review]

**Pull request merged**